test/unit/strkey_test.js

```diff
@@ -0,0 +1,61 @@
+var keypair          = StellarBase.Keypair.master();
+let unencodedBuffer  = keypair.rawPublicKey();
+let unencoded        = unencodedBuffer.toString();
+let accountIdEncoded = keypair.address();
+let seedEncoded      = StellarBase.encodeCheck("seed", unencodedBuffer);
+
+describe('StellarBase#decodeCheck', function() {
+
+  it("decodes correctly", function() {
+    expectBuffersToBeEqual(StellarBase.decodeCheck("accountId", accountIdEncoded), unencodedBuffer);
+    expectBuffersToBeEqual(StellarBase.decodeCheck("seed", seedEncoded), unencodedBuffer);
+  });
+
+  it("throws an error when the version byte is not defined", function() {
```

Comment:
This test could be better if it used a valid base32 encoded value with a bad version byte for the second parameter sent to decode check.  Since we're checking the error string, it's not very important though; otherwise it could lead to false positives since the test does not supply a value that matches only the defined context.